PythonAPI/test/integration_tests_list.txt

```diff
@@ -0,0 +1 @@
+API.spawn_walkers API.spawn_vehicles
```

Comment:
Should we add also collision test here?

Reply:
Not yet as vehicles are not working in UE5. I want to keep the record of the file to add sameones to UE4